docs/content/sidebars/developer.js

```diff
@@ -290,6 +290,14 @@ const developer = [
             'developer/standards/wallet-standard',
         ],
     },
+    {
+        type: 'category',
+        label: 'Tutorials',
+        items: [
+            'developer/tutorials/sponsored-txs',
```

Comment:
```suggestion
            {
                type: 'category',
                label: 'Sponsored Transactions',
                items: [          
                  'developer/tutorials/sponsored-txs',
                  'developer/tutorials/media-platform-package',
                  'developer/tutorials/gas-station-server',
                  'developer/tutorials/transaction-builder',
                ]
             },
```

Please check the syntax and formatting.